src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -45,10 +45,11 @@
      * @throws FormatException when a file has a syntax error
      * @throws PathException   when a file does not exist or is not readable
      */
-    public function load($path/*, ...$paths*/)
+    public function load($path, ...$paths)
```

Comment:
@javiereguiluz see https://github.com/symfony/symfony/pull/23242#discussion_r123096822.
Imho, we could have kept only `... $paths`, maybe throwing if `$paths` is empty.